apps/design/backend/src/worker/generate_election_and_ballot_packages.ts

```diff
@@ -62,7 +70,7 @@ function makeV3Compatible(zip: JsZip, systemSettings: SystemSettings): void {
   );
 }
 
-export async function generateElectionPackage(
+export async function generateElectionAndBallotPackages(
```

Comment:
Just rounding out the renaming I suggested (and potential confusion that may come with "ballot package"), let's call this: `generateElectionPackageAndBallots`